packages/drivers/routerlicious-driver/src/requestUtils.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import Axios, { AxiosRequestConfig, AxiosResponse, AxiosError } from "axios";
+import safeStringify from "json-stringify-safe";
+import { isStatusRetriable, throwR11sNetworkError } from "./r11sError";
+
+export async function enhanceRequestError(error: any): Promise<never> {
+    if (error?.isAxiosError) {
+        const axiosError = error as AxiosError;
+        throwR11sNetworkError(
+            axiosError.response?.statusText
+                ? `${axiosError.response?.data} (${axiosError.response?.statusText})`
+                : axiosError.response?.data,
+            axiosError.response?.status,
+            isStatusRetriable(axiosError.response?.status, axiosError.response?.data?.retryAfter),
```

Comment:
I'd recommend removing canRetry argument - same isStatusRetriable() call is made inside throwR11sNetworkError.
I believe on ODSP side we eventually removed canRetry from similar flow as it was hard to follow code and reason on how to combine value of this flag with status code.
It's better in those cases where you want to overwrite it to just get error object from throwR11sNetworkError and overwrite canRetry property on object - easier to follow code and more obvious on what code does, without complicating throwR11sNetworkError